KnowSheet/scripts/check-headers.sh

```diff
@@ -10,9 +10,16 @@ set -u -e
 CPPFLAGS="-std=c++11 -g -Wall -W -DBRICKS_CHECK_HEADERS_MODE"
 LDFLAGS="-pthread"
 
+if [ $(uname) = "Darwin" ] ; then
+  CPPFLAGS+=" -stdlib=libc++ -x objective-c++ -fobjc-arc"
```

Comment:
To maximize support of c++11 on Mac. In particular, `thread_local` specifier can be used if clang is installed from macports (unfortunately default XCode's clang still doesn't support it)
http://stackoverflow.com/a/23850891